tools/manifest/manifest.py

```diff
@@ -91,6 +91,8 @@ def update(self, tree):
                     hash_changed = True
                 else:
                     new_type, manifest_items = old_type, self._data[old_type][rel_path]
+                if old_type == "reftest" and new_type != old_type:
```

Comment:
This might need to include "reftest_node" too?

Reply:
I actually don't quite understand what "reftest_node" is. Does it contain all of files involved in a reftest, everything that's not used a ref, or something else? What kind of renaming would one have to do to run into a problem here?

Reply:
If we had:

a.html: `<link rel=match href=b.html>`
b.html: `<link rel=match href=c.html>`
c.html: `hello world`

Then: 

a.html would be a reftest (it's a test we want to run)
b.html would be a reftest_node (it's in the reftest graph and can itself be run)
c.html would be a support (because we can't tell anything about it statically)

@jgraham I think that's right?

Reply:
Yeah. I guess it's like theoretically possible you have A==B==C and you update B to be a testharness test but still use it as the ref for A, but that seems pretty unlikely, so this case might not get hit in practice, so it's a bit of an edge case correctness issue. File a bug and mark it "good first bug".

Reply:
I think a less hypothetical case is if b.html changes to `support`? Filing an issue now.

Reply:
https://github.com/w3c/web-platform-tests/issues/9671